test/provider.test.js

```diff
@@ -21,7 +21,7 @@ describe('<RecurlyProvider />', function () {
 
     it('throws an error', function () {
       expect(() => {
-        render(<RecurlyProvider publicKey="test-public-key" />);
+        render(<RecurlyProvider publicKey="test-public-key" api='http://localhost:9877' />);
```

Comment:
I added this localhost address to set the API base url, but none of the tests make assertions against values returned from the server.